app/src/components/common/Header.tsx

```diff
@@ -87,132 +35,81 @@ export const Header = (props: Props) => {
         component="nav"
       >
         <Toolbar sx={{justifyContent: {xs: 'space-between'}}}>
-          <IconButton
-            color="primary"
-            aria-label="open drawer"
-            edge="start"
-            onClick={handleDrawerToggle}
-            sx={{mr: 2, display: {sm: 'none'}}}
-          >
-            <MenuIcon />
-          </IconButton>
           <Stack direction="row" gap={1} sx={{alignItems: 'center'}}>
             <img
               style={{width: '40px', height: '40px'}}
               src={logo}
               alt="Home Unite Us logo"
-              onClick={handleDrawerToggle}
-              role="button"
             />
           </Stack>
           <Stack direction="row" gap={1} sx={{alignItems: 'center'}}>
-            {!user ? (
-              <Box sx={{display: {xs: 'none', sm: 'flex'}}}>
-                {navItems.map(({title, href}) => {
-                  const variant = title === 'Sign Up' ? 'contained' : 'text';
-                  return (
-                    <Button
-                      variant={variant}
-                      href={href}
-                      color="primary"
-                      key={title}
-                    >
-                      {title}
-                    </Button>
-                  );
-                })}
-              </Box>
-            ) : null}
-            {user ? <AvatarDropdownMenu /> : null}
+            <Box sx={{display: {md: 'flex', sm: 'none', xs: 'none'}}}>
+              {navItems.map(({title, href}) => {
+                const variant = title === 'Sign Up' ? 'contained' : 'text';
+                return (
+                  <Button
+                    variant={variant}
+                    href={href}
+                    color="primary"
+                    key={title}
+                  >
+                    {title}
+                  </Button>
+                );
+              })}
+            </Box>
           </Stack>
+          <IconButton
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{height: '40px', width: '40px', display: {md: 'none'}}}
+          >
+            <MenuIcon />
+          </IconButton>
         </Toolbar>
-      </AppBar>
-      <Box component="nav">
-        <Drawer
+        <StyledDrawer
           container={container}
```

Comment:
This hamburger menu does not follow traditional style guidelines. The menu will typically open on the same side of the screen as the button. Our button is on the right but the menu opens on the left. 

I think it looks a lot better to open the menu on the right, but doing this:

```diff
<StyledDrawer>
+  anchor="right"
```

Reply:
common practice! It's been changed to the right.